pages/emailSender.tsx

```diff
@@ -435,120 +357,29 @@ const EmailSender: NextPage = () => {
-              Use customized or standard email subjects?
-            </FormLabel>
-            <FormGroup>
-              <FormControlLabel
-                control={
-                  <Checkbox
-                    onChange={(e) => setCheckedDeliveryBox(e.target.checked)}
-                  />
-                }
-                label="Select custom delivery time"
-              />
-            </FormGroup>
-            {checkedDeliveryBox &&
-            <StyledDateTimeDiv>
-            <LocalizationProvider dateAdapter={AdapterDateFns}>
-              <Stack spacing={3}>
-                <DateTimePicker
-                  label="Select date and time"
-                  value={dateTime}
-                  onChange={(dateTime: Date | null) => {
-                    setDeliveryDateTime(dateTime)
-                  }}
-                  renderInput={(params: any) => <TextField {...params} />}
-                />
-              </Stack>
-            </LocalizationProvider>
-          </StyledDateTimeDiv>}
-            <StyledButton
-              color="info"
-              variant="contained"
-              onClick={() => {
-                handleClickOpen()
-              }}
-              width="medium"
-              disabled={finalMessages.length === 0 || errorMessages.length > 0}
-            >
-              Send!
-            </StyledButton>
-            <Dialog
-              open={open}
-              onClose={handleClose}
-              aria-labelledby="alert-dialog-title"
-            >
-              <DialogTitle id="alert-dialog-title">
-                Are you sure you want to send all emails?
-              </DialogTitle>
-              <DialogActions>
-                <Button variant="contained" onClick={handleClose}>
-                  No
-                </Button>
-                <Button
-                  variant="outlined"
-                  onClick={() => {
-                    handleClose()
-                    sendEmails()
-                  }}
-                  autoFocus
-                >
-                  Yes
-                </Button>
-              </DialogActions>
-            </Dialog>
-            <StyledResultMessage
-              variant="h5"
-              isError={resultMessage.isError}
-            >
-              {resultMessage.message}
-            </StyledResultMessage>
-          </SectionContainer>
+          <SendEmails
+          setCheckedDeliveryBox={setCheckedDeliveryBox}
+          checkedDeliveryBox={checkedDeliveryBox}
+          dateTime={dateTime}
+          handleClickOpen={handleClickOpen}
+          setDeliveryDateTime={setDeliveryDateTime}
+          finalMessagesLength={finalMessages.length === 0}
+          errorMessagesLength={errorMessages.length > 0}
+          handleClose={handleClose}
+          sendEmails={sendEmails}
+          resultMessage={resultMessage}
+          open={open}
+          />
           <StyledFinalMessagesContainer>
```

Comment:
Let's also move 

          <StyledFinalMessagesContainer>
            {displayMessages()}
          </StyledFinalMessagesContainer>

into its own component DisplayMessages or something and then move the StyledFinalMessagesContainer into DisplayMessages